Kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -5,55 +5,36 @@
 
 public class ShapeCollector {
 
-    private List<Shape> figureList = new ArrayList<>();
+    List<Shape> figuresList = new ArrayList<>();
 
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (!(o instanceof ShapeCollector)) return false;
-
-        ShapeCollector that = (ShapeCollector) o;
-
-        return figureList.equals(that.figureList);
-    }
-
-    @Override
-    public int hashCode() {
-        return figureList.hashCode();
-    }
-
-    public List<Shape> addFigure(Shape shape) {
-        if (shape != null) {
-            figureList.add(shape);
-            System.out.println("Figure added.");
+    public int addFigure(Shape shape) {
+        if ( shape != null) {
+            figuresList.add(shape);
         }
-        return figureList;
+        return  figuresList.size();
+
     }
 
     public boolean removeFigure(Shape shape) {
         boolean result = false;
-        if(figureList.contains(shape)) {
-            figureList.remove(shape);
-            System.out.println("Figure removed");
+        if (figuresList.contains(shape)) {
```

Comment:
wysatrczy w tej metodzie sprawdzić czy shape to null (zwrocić false) albo `return shapes.remove(shape);`